mirage/seed_image/catalog_seed_image.py

```diff
@@ -2541,7 +2328,7 @@ def makeGalaxyImage(self, file, psf):
 
         # create attitude matrix so we can calculate the North->V3 angle for
         # each galaxy
-        attitude_matrix = self.getAttitudeMatrix()
+        # attitude_matrix = self.getAttitudeMatrix()
```

Comment:
Below, on line 2344, the `attitude_matrix` variable is being called but its declaration is being commented out here, so that results in an error. Should the `attitude_matrix` be coming from somewhere else?

Reply:
Good catch. That should be `self.attitude_matrix`. I must not have tested with a galaxy input file.